client/src/components/ForgotPassword/style.scss

```diff
@@ -0,0 +1,84 @@
+.login-container {
```

Comment:
why login-container if it is forgot password?
if it is a common class, then choose better name

Reply:
reworked